scripts/primaza.sh

```diff
@@ -56,12 +56,11 @@ function primazaUsage() {
 function build() {
   cmdExec "mvn clean install -DskipTests -Dquarkus.container-image.build=true \
      -Dquarkus.container-image.push=true \
-     -Dquarkus.container-image.registry=${REGISTRY} \
+     -Dquarkus.container-image.registry=127.0.0.1:5000 \
```

Comment:
Si le port défini pour les ENV VAR est de 5001: cfr `REGISTRY=${KIND_REGISTRY:-kind-registry:5001}` pour accéder au registre alors dle numéro du port pour faire un docker pull/push doit être aussi de 5001 ici !

Pq hard codé `-Dquarkus.container-image.registry=127.0.0.1:5000 ` alors qu'en variable ENV est définie `${REGISTRY} ` ?

Reply:
It's ok. I have changed and use the old kind-registry:5000. This was a leftover of several tests I was making...